ui/public/index.html

```diff
@@ -37,7 +37,13 @@
       href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap"
       rel="stylesheet"
     />
-
+    <style>
+      html {
+        height: 100vh;
+        background-image: linear-gradient(to top, #fff 8%, #8E23351e 70%, #fff 100%);
```

Comment:
I presume these colors are consistent with our branding guidelines.

Reply:
Hopefully... it's a semi-transparent shade of "merlot" from https://agoric.com/brand/. Any solid colors would be too jarring though imo. This is probably the boldest style change I'll make without product/UX input.